mmv1/products/developerconnect/Connection.yaml

```diff
@@ -0,0 +1,183 @@
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/connections
+create_url: projects/{{project}}/locations/{{location}}/connections?connectionId={{connection_id}}
+self_link: projects/{{project}}/locations/{{location}}/connections/{{connection_id}}
+id_format: projects/{{project}}/locations/{{location}}/connections/{{connection_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/connections/{{connection_id}}
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: "developer_connect_connection_basic"
+    primary_resource_id: "my-connection"
+    primary_resource_name: "fmt.Sprintf(\"tf-test-connection%s\",
+      context[\"random_suffix\"\
+      ])"
+    vars:
+      connection_name: "tf-test-connection"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: "developer_connect_connection_github_doc"
+    skip_test: true
+name: Connection
+description: "A connection to a GitHub App installation."
+min_version: "beta"
+update_verb: :PATCH
+update_mask: true
+autogen_async: true
+properties:
+  - !ruby/object:Api::Type::NestedObject
+    properties:
```

Comment:
can you place the properties last, so the other attributes can be read first. this will make it inline with other YAML and also not break a script we have